src/linear_algebra/affine_space/affine_subspace.lean

```diff
@@ -7,19 +7,13 @@ import algebra.add_torsor
 import linear_algebra.tensor_product
 import data.set.intervals.unordered_interval
 
-noncomputable theory
-open_locale big_operators
-open_locale classical
-
 /-!
 # Affine spaces
 
-This file defines affine spaces (over modules) and subspaces, affine
-maps, and the affine span of a set of points.  For affine combinations
-of points, see `linear_algebra.affine_space.combination`.  For
-affinely independent families of points, see
-`linear_algebra.affine_space.independent`.  For some additional
-results relating to finite-dimensional subspaces of affine spaces, see
+This file defines affine spaces (over modules) and subspaces, and the affine span of a set of
```

Comment:
A similar comment applies here now this file is no longer defining `affine_space.